libs/sdk-ui-filters/src/AttributeFilter/utils/AttributeFilterUtils.ts

```diff
@@ -0,0 +1,29 @@
+// (C) 2021 GoodData Corporation
+
+import { IAttributeElement } from "@gooddata/sdk-backend-spi";
+
+export const getAllTitleIntl = (intl: any, isInverted: boolean, empty: boolean, equal: boolean) => {
+    if ((isInverted && empty) || (!isInverted && equal)) {
+        return intl.formatMessage({ id: "attrf.all" });
+    }
+    return intl.formatMessage({ id: "attrf.all_except" });
+};
+
+export const getNoneTitleIntl = (intl: any) => {
+    return intl.formatMessage({ id: "gs.filterLabel.none" });
+};
+
+export const getItemsTitles = (selectedFilterOptions: IAttributeElement[]) => {
+    return selectedFilterOptions.map((selectedOption) => selectedOption.title).join(", ");
+};
+
+export const truncateTitle = (title: string, length?: number, ending?: string) => {
```

Comment:
There is a function in @gooddata/util for this called `shortenText` that does the same thing, Consider using that.

Reply:
Used shortenText as truncateTitle with custom length and ending settings is never used.